app/controllers/dams_resource_controller.rb

```diff
@@ -30,6 +30,17 @@ def show
     # get metadata from solr
     @document = get_single_doc_via_search(1, {:q => "id:#{params[:id]}"} )
 
+    # generate facet collection list for collection page only
+    if !@document.nil?
```

Comment:
I've updated it for status 404. See commit https://github.com/ucsdlib/damspas/commit/78d697388089371ed486a4bcc4500dc9fa47c9a2 .